homeassistant/components/velbus/__init__.py

```diff
@@ -132,6 +135,42 @@ async def set_memo_text(call: ServiceCall) -> None:
         ),
     )
 
+    async def clear_cache(call: ServiceCall) -> None:
+        """Handle a clear cache service call."""
+        # clear the cache
+        with suppress(FileNotFoundError):
+            if call.data[CONF_ADDRESS]:
+                await hass.async_add_executor_job(
+                    os.unlink,
+                    hass.config.path(
+                        STORAGE_DIR,
+                        f"velbuscache-{call.data[CONF_INTERFACE]}/{call.data[CONF_ADDRESS]}.p",
```

Comment:
This cache is not the same as the cache that uses the config entry id to deduplicate?

Reply:
Yes it is the same

Reply:
Ok